lib/takeaway_menu.rb

```diff
@@ -0,0 +1,22 @@
+class Menu
+
+  def menu
```

Comment:
You could simply have one public method `menu` returning the hash directly, delegating to a private method is not necessary